sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -162,6 +162,23 @@ function isSubsumedByHigherPath(normalizedPath: string, normalizedPathSet: Set<s
     return false;
 }
 
+/**
+ * searchUp searches for and returns the parent directory path
```

Comment:
I'd remove the reference to "parent" here
```suggestion
 * searchUp searches for and returns the first directory path
```